### ISSUE TITLE
Version 12.7.6 of macOS Monterey is missing

### ISSUE THREAD

**Issue by Asnowz** [ninxsoft/Mist#156]
### :beetle: Description
In the download list, the latest version displayed is 12.7.4, whereas 12.7.6 has been released.

### :computer: Environment
- Mist version: **0.20.1**
- macOS Version (`sw_vers`): **15.1**

### :camera: Screenshots
<img width="819" alt="ScreenFloat Shot of App Store at 22 Nov 2024 at 12_07_06" src="https://github.com/user-attachments/assets/53396dee-48a5-4ee9-836d-a35fb8db3ddb">
<img width="466" alt="ScreenFloat Shot of Mist at 22 Nov 2024 at 12_06_40" src="https://github.com/user-attachments/assets/2fff6e23-e9d7-426a-884a-b1f71433205a">

**Comment by wakco** [comment]
Not a bug, Apple have not provided a newer full installer for macOS 12 Monterey than version 12.7.4. Even the ipsw files for Apple Silicon stop at 12.6.

**Comment by Asnowz** [comment]
Yes, I can confirm that I was able to download Monterey 12.7.6 with Mist before.

**Comment by Chromeking** [comment]
Has this been resolved or did I miss something with regards to this?  My panel is only showing Monterey 12.4.xxx

**Comment by brandon-fryslie** [comment]
Seems to be resolved as far as this project is concerned.  Apple does not provide the installer.

> Not a bug, Apple have not provided a newer full installer for macOS 12 Monterey than version 12.7.4. …
> 
> If you would like to confirm this, try softwareupdate --list-full-installers at the command line
